apps/journeys-admin/src/components/JourneyList/JourneyCard/JourneyCard.tsx

```diff
@@ -12,90 +13,82 @@ interface JourneyCardProps {
-      <Typography
-        variant="subtitle1"
-        sx={{
-          display: '-webkit-box',
-          WebkitLineClamp: 2,
-          WebkitBoxOrient: 'vertical',
-          overflow: 'hidden'
-        }}
-        gutterBottom
-      >
-        {journey.title}
-      </Typography>
+    <Card sx={{ borderRadius: '0px', px: 6, py: 4 }}>
+      <Link href={`/journeys/${journey.slug}`} passHref>
+        <Typography
+          variant="subtitle1"
+          component="div"
+          noWrap
+          gutterBottom
+          sx={{ color: (theme) => theme.palette.secondary.main }}
+        >
+          {journey.title}
+        </Typography>
+      </Link>
 
       <Typography
-        variant="body2"
+        variant="caption"
+        noWrap
         sx={{
-          display: '-webkit-box',
-          WebkitLineClamp: 2,
-          WebkitBoxOrient: 'vertical',
-          overflow: 'hidden'
+          display: 'block',
+          color: (theme) => theme.palette.secondary.main
         }}
         gutterBottom
       >
-        {date}
+        {formattedDate}
         {journey.description !== null && ` - ${journey.description}`}
       </Typography>
 
-      <Box sx={{ display: 'flex' }}>
-        {journey.status === 'draft' ? (
-          <Chip
-            label={'Draft'}
-            icon={<EditIcon style={{ color: '#F0720C' }} />}
-            sx={{
-              height: '42px',
-              width: 'auto',
-              borderRadius: '18px',
-              fontSize: '17px',
-              backgroundColor: 'white',
-              ml: 4
-            }}
-          />
-        ) : (
-          <Chip
-            label={'Published'}
-            icon={<CheckCircleIcon style={{ color: '#3AA74A' }} />}
-            sx={{
-              height: '42px',
-              width: 'auto',
-              borderRadius: '18px',
-              fontSize: '17px',
-              backgroundColor: 'white',
-              ml: 4
-            }}
-          />
-        )}
-        {/* Locale formatting does not currently work for every locale */}
-        <Chip
-          label={`${journey.locale
-            .substring(0, 2)
-            .toUpperCase()} (${journey.locale.substring(3)})`}
-          icon={<TranslateIcon />}
-          sx={{
-            height: '42px',
-            width: 'auto',
-            borderRadius: '18px',
-            fontSize: '17px',
-            backgroundColor: 'white',
-            ml: 4
-          }}
-        />
+      <Box sx={{ display: 'flex', alignItems: 'center' }}>
+        <Grid container>
+          <Grid item>
+            {journey.status === 'draft' ? (
+              <EditIcon
+                sx={{
+                  color: (theme) => theme.palette.warning.main,
+                  pr: '6px'
+                }}
+              />
+            ) : (
+              <CheckCircleIcon
+                sx={{
+                  color: (theme) => theme.palette.success.main,
```

Comment:
```suggestion
                  color: 'success.main',
```
And remove the same in every other place you use it